modules/firehose/README.md

```diff
@@ -1,5 +1,5 @@
 # Firehose Module
-Firehose module is designed to support CloudWatch metrics and logs.
+Firehose module is designed to support CloudWatch metrics.
```

Comment:
I'm not sure why not

Reply:
The firehose integration in terraform send only metrics to coralogix and not logs